hack/godep-save.sh

```diff
@@ -31,4 +31,5 @@ REQUIRED_BINS=(
 
 pushd "${KUBE_ROOT}" > /dev/null
   GO15VENDOREXPERIMENT=1 ${GODEP} save "${REQUIRED_BINS[@]}"
+  GO15VENDOREXPERIMENT=1 ${GODEP} save "${REQUIRED_BINS[@]}"
```

Comment:
Is this for temporary walkaround? If so, maybe we need some comments here to explain or add a TODO.